### ISSUE TITLE
No release tagged in 2018

### ISSUE THREAD

**Issue by Duffyx** [wting/autojump#556]
Hello, I saw that the latest release that is tagged in the repo dates from 2017. Are you working towards a release version to tag one again? This is useful for frameworks that want to import your tool in their package repository.

**Comment by ankurloriya** [comment]
Yes, I could not found data in following API https://api.github.com/repos/wting/autojump/releases, Using this url I want to make a script for auto install tools when I reinstall my ubuntu
